src/program.rs

```diff
@@ -155,6 +184,7 @@ impl Program {
                     Action::ReRender => self.render()?,
                     Action::Next => self.increment(1)?,
                     Action::Prev => self.decrement(1)?,
+                    Action::Copy => self.copy_image()?,
```

Comment:
Can you match on this in the same way as `move_image`? Otherwise some error in copying the image will crash the program.

Reply:
Yep!